azurerm/internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -61,6 +66,84 @@ func resourceMsSqlVirtualMachine() *schema.Resource {
 				}, false),
 			},
 
+			"auto_backup": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"backup_system_databases": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+
+						"backup_schedule_automated": {
```

Comment:
if we are not going to have enabled here this should be
```suggestion
						"automated_backup_schedule": {
```

and does this preclude setting the vull backup schedule properties

Reply:
It does preclude setting the scheduling properties, putting those in a block will help tidy that up.